crates/ra_hir/src/ty/tests.rs

```diff
@@ -3287,3 +3287,107 @@ fn no_such_field_diagnostics() {
 "###
     );
 }
+
+#[cfg(test)]
```

Comment:
That's already inside `tests`, no need to `cfg` further

Reply:
Fixed.